### DISCUSSION TITLE
Collapsable code blocks?

### DISCUSSION THREAD

**Discussion by 1coderookie** [squidfunk/mkdocs-material#5082] [Q&A] [answered]
Hi everybody,
I wonder if there's a solution to make code blocks collapsible somehow, so that they're still displayed with the regular font size but only the first few lines appear unless you click on expand.  
I tried with admonitions (and also with the `<details> ...codeblock... </details> solution which turns into an admonition box as well) but due to the smaller size of the font (which especially comes into account on mobile devices) I don't really like to use it for bigger code blocks.  
Any idea?  
Thanks!

**Comment by squidfunk** [comment, marked as answer]
Duplicate of: 

- #4964
- #3444

**Reply by 1coderookie** [reply]
Oh my - I'm so sorry, I didn't see that one.. :(
Thanks!
